libsolidity/interface/CompilerStack.cpp

```diff
@@ -1206,13 +1206,69 @@ Json CompilerStack::ethdebug(Contract const& _contract, bool _runtime) const
 	solAssert(m_stackState >= AnalysisSuccessful, "Analysis was not successful.");
 	solAssert(_contract.contract);
 	solUnimplementedAssert(!isExperimentalSolidity());
-	if (_runtime)
+	Json result = {
+	{"contract",
+		{
+			{"name", _contract.contract->name()},
+			{"definition",
+				{{"sources",
+					{{"id", static_cast<int>(sourceIndices()[_contract.contract->sourceUnitName()])}}
+				}}
+			}}
+		},
+		{"environment ", _runtime ? "call" : "create"},
+		{"instructions", ethdebugInstructions(_contract, _runtime)}
+	};
```

Comment:
This is a bit weird to format but I think this here might be a bit closer to our style guides:

```suggestion
	Json result = {
		{
			"contract",
			{
				{"name", _contract.contract->name()},
				{
					"definition",
					{{
						"sources",
						{{"id", static_cast<int>(sourceIndices()[_contract.contract->sourceUnitName()])}}
					}}
				}
			}
		},
		{"environment ", _runtime ? "call" : "create"},
		{"instructions", ethdebugInstructions(_contract, _runtime)}
	};
```